doc/backups.rst

```diff
@@ -72,4 +72,64 @@ To secure the user, use an extemely restricted permission in ~/.ssh/authorized_k
 Restoring
 ---------
 
-TODO
+How to backup production servers:
+=================================
+Our backup process consists of two steps:
+1- Our current backup script backs up the production instance in the borg_backups repository located in the project path.
+
+2- Afterwards, this borg repository is copied to an ftp server associated to each production instance.
+
+This FTP server is a free service provided by OVH for each dedicated server and must be manually instanciated via the OVH manager.
+
+TODO: Automate enabling the ftp backup server via OVH APIs.
+
+In order to manually backup a production instance, launch the following::
+
+    fab -c {RC_FILE} execute_backup_borg_repository
+
+In order to set a crontab for a production instance, use this command::
+
+    fab -c {RC_FILE} cron_backup_borg_repository
+
+
+These two commands use the following pattern:
+1. Make sure borg is installed.
+2. Put a configuration file on the remote host with the server FTP information.
+3. Creates a backup script from template with the appropriate remote host information and puts it on the remote host.
+4. For the second host, it sets a cron job to automatically backup the instance daily.
+
+
+How to list backups on a production server:
+===========================================
+Use the command::
+
+    fab -c {RC_FILE} list_backups
+
+How to restore a backup on a production instance:
+=================================================
+Use the command::
+
+    fab -c {RC_FILE} fetch_backup
+
+This command will fetch the backup from the ftp server. If you don't fix the name,
+by default it will get the last backup folder.
+
+You will have the backup assembl in a newly created `home` folder.
+
+The command `fetch_backup` will also move the backup up assembl to a newly created
+folder: assembl_backup.
```

Comment:
should it be inside assembl root?

Reply:
not sure I understand what you mean

Reply:
You are presumably working from within an assembl root since you have access to the fabfile.
Then you are creating a new *nested* assembl root folder called assembl_backup, from within that root. Nested assembl folders are a bad smell. I think you should either explain how to choose another folder, or how to download the fabfile independently of assembl, so that you're not starting from within an assembl folder.

Reply:
the folder is supposed to be `/home/assembl_user/assembl_backup/assembl/` . in the doc, I wrote that the path for the new assembl will be `/home/assembl_user/assembl_backup/assembl/`.  so I change it inn the backup_rc_file  before restoring

Reply:
My problem is : you start the "restore" operation description with a fab command. Where are you? In a pristine server, or in an existing assembl installation? If the former, how should there be a fabfile available at this point? You do not explain how to fetch it. If the latter, you're creating assembl_backup from within an existing assembl installation, which is a bad practice. We lack context for this section.

Reply:
It is actually the former case, the fabfile is on my local machine, I execute my commands using the backup_rc file just with changing the projectpath. Basically, we are setting another assembl on the machine